def my_function(fname, lname):#crea la funcvion y adentro esta fname y Lname
  print(fname + " " + lname)#coloca lo pedido

my_function("Emil")#el argumento es uno asi que da error
